changelogs/fragments/69935-2.10-deprecation-support.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+ - "``Display.deprecated()``, ``AnsibleModule.deprecate()`` and ``Ansible.Basic.Deprecate()`` now also accept the deprecation-by-date parameters and collection name parameters from Ansible 2.10. This prevents collections plugins and modules from Ansible 2.10 conforming collections to crash with newer versions of Ansible 2.9."
```

Comment:
```suggestion
 - "``Display.deprecated()``, ``AnsibleModule.deprecate()`` and ``Ansible.Basic.Deprecate()`` now also accept the deprecation-by-date parameters and collection name parameters from Ansible 2.10, so plugins and modules in collections that conform to Ansible 2.10 will run with newer versions of Ansible 2.9."
```

Reply:
@acozine thanks a lot, that's definitely better!